src/lib/childrenUtils.js

```diff
@@ -16,3 +16,14 @@ export const someByType = (children, type) => _.some(Children.toArray(children),
  * @returns {undefined|Object}
  */
 export const findByType = (children, type) => _.find(Children.toArray(children), { type })
+
+/**
+ * Tests if any children.
+ * @param {Object} children The children prop of a component.
+ * @returns {Boolean}
+ */
+export const isNil = (children) => {
+  if (children == null) return true // eslint-disable-line
+  if (Array.isArray(children) && children.length == 0) return true // eslint-disable-line
```

Comment:
The Lodash implementation uses `==`, the deeper `===` actually breaks things.  Suggestion?